indy_node/test/api/helper.py

```diff
@@ -177,10 +177,42 @@ def validate_claim_def_txn(txn):
 
 # Misc utility
 
-def sdk_write_schema(looper, sdk_pool_handle, sdk_wallet_steward):
+def sdk_write_schema(looper, sdk_pool_handle, sdk_wallet_steward, multi_attribute=[], name="", version=""):
     _, identifier = sdk_wallet_steward
-    _, schema_json = looper.loop.run_until_complete(
-        issuer_create_schema(identifier, "name", "1.0", json.dumps(["first", "last"])))
+
+    if multi_attribute:
```

Comment:
We should probably write these methods with same style as in https://github.com/hyperledger/indy-node/pull/1034